cli/package.json

```diff
@@ -39,7 +39,6 @@
     "rimraf": "^3.0.2",
     "semver": "^7.3.2",
     "table": "^5.4.6",
-    "through": "^2.3.8",
```

Comment:
Can you also remove `through_vx.x.x.js` if you're removing this dep?

Reply:
Sure. I had removed `through_vx.x.x.js`.

Reply:
I recommended that @HsuTing remove the lib def because `through` is not used in the cli codebase anymore. So it makes sense to also remove `through` as a lib def in `cli/flow-typed/npm` This is unrelated to lib defs in `definitions`

Reply:
I had remove `through_vx.x.x.js` again. Because updating branch, `through` appears again.

Reply:
Ah, I wasn't paying attention to the path name, thanks for the catch!